.github/workflows/push.yml

```
@@ -88,8 +88,8 @@ jobs:
       - name: Set outputs
         id: set-outputs
         run: |
-          echo ${{ env.publisher }}
-          echo "publisher=${{ env.publisher }}" >> $GITHUB_OUTPUT
+          echo "${{ env.publisher }}""
```

I don't think we want two escape quotes here

---

Yeah this looks like a typo
```suggestion
          echo "${{ env.publisher }}"
```

---

Yeah was a type